example/extensions/lib_custom_op/README.md

```diff
@@ -56,87 +72,111 @@ For building a library containing your own custom operator, compose a C++ source
 - `forward` - Forward Computation (can be replace with `createOpState`, see below for details)
 
 Then compile it to `libmyop_lib.so` dynamic library using the following command:
+
 ```bash
 g++ -shared -fPIC -std=c++11 myop_lib.cc -o libmyop_lib.so -I ../../../include/mxnet
 ```
 
+If you don't want to download MXNet source and choose only using `lib_api.h` header, you can copy the header over to the same folder of `myop_lib.cc` and run:
+
+```bash
+g++ -shared -fPIC -std=c++11 myop_lib.cc -o libmyop_lib.so
+```
+
 Finally, you can write a Python script to load the library and run your custom operator:
```

Comment:
Finally, you can write some code to load the library and run your custom operator in any language binding. Heres an example in Python (but C Predict API and C++ APIs work too):